README.md

```diff
@@ -40,6 +40,12 @@ Ultimately, your directory structure should look like this:
 
 ## Build
```

Comment:
I believe this steps assumes that you've completed the previous step (i.e [Setup](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/master/README.md#setup))

Reply:
@mihirsoni : Yes you're right. I missed that. Sorry.